### PR TITLE
Get multiport-eswitch mode from metalnet rundir

### PR DESCRIPTION
**Pull request by PlagueCZ**
In OSC it is beneficial to use the same metalnet deployment for both versions of dpservice - legacy (single-port) and multiport-eswitch ones.

Thus using the command-line argument is not the way to go, instead we simply `echo -n "eswitch" > /var/run/metalnet/mode` and metalnet then reads this information, just like it gets other stuff from virtlet(?) about VFs.

The original implementation in 78efc7ccb9313819ab603e36b4252342f34a29e8 is tested in our clusters, I simplified the code without being able to run it in a cluster, though I did verify the output locally.

### CODE REVIEW

**Review by byteocean — APPROVED** [review]
Looks no issue to me. But it would be nice to have one line comment like: run `touch /var/run/metalnet/mode && echo -n "eswitch" > /var/run/metalnet/mode` to enable multiport eswitch mode, as this approach of configuration is not so crystal clear it is a manual step to setup the flag.

**Comment by PlagueCZ** [comment]
I added a comment and also some documentation in README.md covering multiport-eswitch and pf1-proxy

**Comment by PlagueCZ** [comment]
I have reverted the removal of `--multiport-eswitch` and mentioned the automatic detection in help and markdown.

File: main.go [hunk 1]
```diff
@@ -104,7 +104,7 @@ func main() {
 	flag.BoolVar(&enableIPv6Support, "enable-ipv6", false, "Enable IPv6 support")
 	flag.IntVar(&publicVNI, "public-vni", 100, "Virtual network identifier used for public routing announcements.")
 	flag.IPVar(&routerAddress, "router-address", net.IP{}, "The address of the next router.")
-	flag.BoolVar(&multiportEswitchMode, "multiport-eswitch", false, "Enable multiport eswitch support")
+	flag.BoolVar(&multiportEswitchMode, "multiport-eswitch", false, "Enable multiport eswitch support (unless specified in metalnet-dir)")
```
[the remark this answers is not in the document]
Comment (PlagueCZ) [reply main.go:107]: Well arguments are already explicit, but then we would say "explicit unless" which is strange, so how about changing the parentheses to "(file in metalnet-dir can override this)"?

[the remark this answers is not in the document]
Comment (PlagueCZ) [reply main.go]: I have reworded the help and also changed the handling so it should work normally when the metalnet-dir file is not present

**Review by guvenc — APPROVED** [review]
LGTM

**Pull request merged**